data/BR.yml

```diff
@@ -0,0 +1,17 @@
+# +additions_only+ will tell business_calendar to not attempt to fetch holidays from the underlying
+# `holidays` gem, and instead only use the defined +addtions+ here. Useful e.g. for cases where a
```

Comment:
While you're here, could you fix the `additions` typo that's been replicated into N files now?